src/test/java/com/rewedigital/composer/routing/RoutingConfigurationTest.java

```diff
@@ -29,22 +35,42 @@ public void allConfigParametersAreCoveredByDefaultConfig() {
     @Test
     public void createsLocalRouteFromConfiguration() {
         final RoutingConfiguration configuration =
-            RoutingConfiguration.fromConfig(configWithLocalRoutes(
-                localRoute("/test/path/<arg>", "GET", "https://target.service/{arg}", RouteTypeName.PROXY)));
+            RoutingConfiguration.fromConfig(configWithSingleLocalRoute(withoutTtl));
         final List<Rule<Match>> localRules = configuration.localRules();
         assertThat(localRules).anySatisfy(rule -> {
             assertThat(rule.getPath()).isEqualTo("/test/path/<arg>");
             assertThat(rule.getMethods()).contains("GET");
-            assertThat(rule.getTarget()).isEqualTo(Match.of("https://target.service/{arg}", RouteTypeName.PROXY));
+            final Match routeMatchWithoutTtl =
+                Match.of("https://target.service/{arg}", Optional.empty(), PROXY);
+            assertThat(rule.getTarget()).isEqualTo(routeMatchWithoutTtl);
         });
     }
 
+    @Test
+    public void setsTtlIfConfigures() {
+        final RoutingConfiguration configuration =
+            RoutingConfiguration.fromConfig(configWithSingleLocalRoute(withTtl));
+        final List<Rule<Match>> localRules = configuration.localRules();
+        assertThat(localRules).anySatisfy(rule -> {
+            final Match routeMatchWithTtl =
+                Match.of("https://target.service/{arg}", Optional.of(ofMillis(200)), PROXY);
```

Comment:
I think i don't like the static import of `Duration.ofMillis`, because `Duration.ofMills(200)` makes more sense to me than just `ofMillis(200)`